packages/e2e-testing/__tests__/e2e-utils.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as fs from 'fs';
+
+import axios, {AxiosResponse} from 'axios';
+import _ from 'lodash';
+import {Wallet as ChannelWallet} from '@statechannels/server-wallet';
+import {ChannelResult} from '@statechannels/client-api-schema';
+import {providers, Contract} from 'ethers';
+import {ContractArtifacts} from '@statechannels/nitro-protocol';
+import {BN, NULL_APP_DATA} from '@statechannels/wallet-core';
+import {Logger} from '@graphprotocol/common-ts';
+
+import {createTestLogger, generateAllocationIdAndKeys} from '../src/utils';
+
+export function setupLogging(logFilePath: string): Logger {
+  logFilePath && fs.existsSync(logFilePath) && fs.truncateSync(logFilePath);
+  const logger = createTestLogger(logFilePath);
+  (logger as any).level = 'debug';
+  return logger;
+}
+
+let provider: providers.JsonRpcProvider;
+const mine6Blocks = () => _.range(6).forEach(() => provider?.send('evm_mine', []));
+
+export function setupContractMonitor(rpcEndpoint: string, ethAssetHolderAddress: string): Contract {
+  provider = new providers.JsonRpcProvider(rpcEndpoint);
```

Comment:
This suggestion should not be applied from GH.

```suggestion
const mine6Blocks = (provider: providers.JsonRpcProvider) => _.range(6).forEach(() => provider.send('evm_mine', []));

export function setupContractMonitor(provider: providers.JsonRpcProvider, rpcEndpoint: string, ethAssetHolderAddress: string):  {
```
```